### DISCUSSION TITLE
Matek F405-WMN will not disarm

### DISCUSSION THREAD

**Discussion by 1Steven** [iNavFlight/inav#9657] [Q&A] [answered]
Hello all,
I've run into a perplexing problem. My Matek F405-WMN will not disarm. I'm using MP and have all arming 0'd out but it still arms. QGC is the same. The disarm command in MP is rejected and so is the forced disarm command. Everything else is working perfectly, including Yaapu, Avatar OSD and data to MP. I can try reflashing with the latest but wanted to see if anyone here had some ideas first.
Thanx in advance.

**Comment by MrD-RC** [comment]
INAV is not made to work with Mission Planner or QGC. They are ArduPilot GCS. There is some MAVLink functionality. But INAV was never made woth QGC or Mission Planner in mind. 

You should use an MSP based GCS like MWPTools.

**Reply by 1Steven** [reply]
Thanx. My bad, I didn’t mention I’m using Arduplane, not INAV. I’m very familiar with MP, QGC and Ardupilot.

**Reply by MrD-RC** [reply, marked as answer]
This is an INAV repository. It has nothing to do with ArduPilot. You probably want to be asking this question on the ArduPilot repository if that’s what you’re using.

**Reply by 1Steven** [reply]
Oh, my apologies. My first use. 
Thanx.
